editor/svgedit.js

```diff
@@ -8,21 +8,23 @@
 svgedit = {
 	// common namepaces constants in alpha order
 	NS: {
-		HTML: 'http://www.w3.org/1999/xhtml',
-		MATH: 'http://www.w3.org/1998/Math/MathML',
-		SE: 'http://svg-edit.googlecode.com',
-		SVG: 'http://www.w3.org/2000/svg',
-		XLINK: 'http://www.w3.org/1999/xlink',
-		XML: 'http://www.w3.org/XML/1998/namespace',
-		XMLNS: 'http://www.w3.org/2000/xmlns/' // see http://www.w3.org/TR/REC-xml-names/#xmlReserved
+		html: 'http://www.w3.org/1999/xhtml',
```

Comment:
I don't understand why these needed to be changed.  Conventionally uppercase things are constants (which namespaces are)